app/controllers/user.js

```diff
@@ -54,3 +82,29 @@ exports.signUp = ({ user }, res, next) =>
         );
     })
     .catch(next);
+
+exports.usersPerPage = 2;
+
+exports.listUsers = (req, res, next) => {
+  try {
+    const decoded = jwt.decode(req.headers.token);
```

Comment:
Why decode token? Remove this. decoded never used

Reply:
user list card states: -Se debe estar autenticado para consumir dicho recurso.
though constant decoded is never used (and will be removed), decoded must be performed to compare the token against the api secret